spec/apm-cli-spec.coffee

```diff
@@ -25,6 +27,11 @@ describe 'apm command line interface', ->
       callback = jasmine.createSpy('callback')
       apm.run(['-v', '--no-color'], callback)
 
+      testAtomVersion = '0.0.0'
+      tempAtomResourcePath = temp.mkdirSync('apm-resource-dir-')
+      fs.writeFileSync(path.join(tempAtomResourcePath, 'package.json'), JSON.stringify(version: testAtomVersion))
+      process.env.ATOM_RESOURCE_PATH = tempAtomResourcePath
```

Comment:
This is reset at the end of the test, correct?  If not it needs to be.

Reply:
When you say this, do you mean `process.env.ATOM_RESOURCE_PATH`? 

If so, I am basing this test logic off of other `-spec` examples. I do not see where it is reset. Is there an example of how to reset it?

https://github.com/atom/apm/blob/5cf83aacbff6ac0e31d6a760974ed9fdb23f67a9/spec/list-spec.coffee#L38-L47
https://github.com/atom/apm/blob/c9f021038f847376bff53150f1e2b28362b9981b/spec/install-spec.coffee#L24-L25
https://github.com/atom/apm/blob/d55f94ad9a77f21681becc9196947b5f53fca83d/spec/upgrade-spec.coffee#L143-L144
https://github.com/atom/apm/blob/d399d88c47ed4f54f455419674e05371aa44c0e5/spec/publish-spec.coffee#L19-L22
https://github.com/atom/apm/blob/b7586287912dd3996a0aed9e1ab025a03d8c674d/spec/rebuild-spec.coffee#L30-L35

Reply:
Ah ok, it is reset.  You linked to it in the first code snippet.  https://github.com/atom/apm/blob/5cf83aacbff6ac0e31d6a760974ed9fdb23f67a9/spec/list-spec.coffee#L45